maven/impl-maven/src/test/java/org/jboss/shrinkwrap/resolver/api/maven/coordinate/MavenDependenciesTestCase.java

```diff
@@ -1,19 +1,3 @@
-/*
```

Comment:
Why was this removed?

Reply:
This was not intentional, probably some random input from my keyboard... Thanks for noticing it.